src/Common/src/Interop/Unix/libssl/SslConnectionInfo.cs

```diff
@@ -26,7 +26,8 @@ internal SslConnectionInfo(SafeSslHandle sslContext)
                 out DataCipherAlg,
                 out KeyExchangeAlg,
                 out DataHashAlg,
-                out DataKeySize))
+                out DataKeySize,
+                out DataHashKeySize))
             {
                 throw Interop.OpenSsl.CreateSslException(SR.net_ssl_get_connection_info_failed);
             }
```

Comment:
Seems openssl doesnot provide a way to return a exchangekey size.. in dtls1_send_server_key_exchange  and similar function for other sslProtocols , it internally calculates the key exchange size and generate key.. It is not a constant either that we can hardcode and return.